src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -396,31 +407,31 @@ public partial class XmlElement : System.Xml.XmlLinkedNode
         public override System.Xml.XmlNode CloneNode(bool deep) { throw null; }
         public virtual string GetAttribute(string name) { throw null; }
-        public virtual string GetAttribute(string localName, string namespaceURI) { throw null; }
-        public virtual System.Xml.XmlAttribute GetAttributeNode(string name) { throw null; }
-        public virtual System.Xml.XmlAttribute GetAttributeNode(string localName, string namespaceURI) { throw null; }
+        public virtual string GetAttribute(string localName, string? namespaceURI) { throw null; }
+        public virtual System.Xml.XmlAttribute? GetAttributeNode(string name) { throw null; }
+        public virtual System.Xml.XmlAttribute? GetAttributeNode(string localName, string? namespaceURI) { throw null; }
         public virtual System.Xml.XmlNodeList GetElementsByTagName(string name) { throw null; }
         public virtual System.Xml.XmlNodeList GetElementsByTagName(string localName, string namespaceURI) { throw null; }
         public virtual bool HasAttribute(string name) { throw null; }
-        public virtual bool HasAttribute(string localName, string namespaceURI) { throw null; }
+        public virtual bool HasAttribute(string localName, string? namespaceURI) { throw null; }
         public override void RemoveAll() { }
         public virtual void RemoveAllAttributes() { }
         public virtual void RemoveAttribute(string name) { }
-        public virtual void RemoveAttribute(string localName, string namespaceURI) { }
-        public virtual System.Xml.XmlNode RemoveAttributeAt(int i) { throw null; }
-        public virtual System.Xml.XmlAttribute RemoveAttributeNode(string localName, string namespaceURI) { throw null; }
-        public virtual System.Xml.XmlAttribute RemoveAttributeNode(System.Xml.XmlAttribute oldAttr) { throw null; }
+        public virtual void RemoveAttribute(string localName, string? namespaceURI) { }
+        public virtual System.Xml.XmlNode? RemoveAttributeAt(int i) { throw null; }
+        public virtual System.Xml.XmlAttribute? RemoveAttributeNode(string localName, string? namespaceURI) { throw null; }
+        public virtual System.Xml.XmlAttribute? RemoveAttributeNode(System.Xml.XmlAttribute oldAttr) { throw null; }
         public virtual void SetAttribute(string name, string value) { }
-        public virtual string SetAttribute(string localName, string namespaceURI, string value) { throw null; }
-        public virtual System.Xml.XmlAttribute SetAttributeNode(string localName, string namespaceURI) { throw null; }
+        public virtual string SetAttribute(string localName, string? namespaceURI, string value) { throw null; }
+        public virtual System.Xml.XmlAttribute SetAttributeNode(string localName, string? namespaceURI) { throw null; }
         public virtual System.Xml.XmlAttribute SetAttributeNode(System.Xml.XmlAttribute newAttr) { throw null; }
```

Comment:
As per [documentation](https://docs.microsoft.com/dotnet/api/system.xml.xmlelement.setattributenode?view=net-5.01):
>If the attribute replaces an existing attribute with the same name, the old XmlAttribute is returned; otherwise, null is returned.
```suggestion
        public virtual System.Xml.XmlAttribute? SetAttributeNode(System.Xml.XmlAttribute newAttr) { throw null; }
```

Reply:
I don't believe this actually returns null - looking at the implementation it returns old node if it already exists and new node if it doesn't

Reply:
I changed it because is `virtual` and someone could override it to return `null`; looking at the reference within the code, we don't even dereference the method's return value so it doesn't really affect libraries code if the method returns `null` or not.

If you feels like is pointless to make it nullable I can undo this change.